### ISSUE TITLE
missing `/etc/machine-id` causes compatibility issues

### ISSUE THREAD

**Issue by MrBuddyCasino** [docker/for-mac#3554]
- [X] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] I have uploaded Diagnostics

### Expected behavior

The file `/etc/machine-id` should be present and not be empty.

### Actual behavior

The file `/etc/machine-id` is missing.

### Information
Docker for Mac 2.0.0.3 
macOS 10.14.3 (18D109)

### Steps to reproduce the behavior
Try to mount it in a K8S yaml, it will fail:
```
          volumeMounts:
            - name: machine-id
              mountPath: /etc/machine-id
```

**Comment by MrBuddyCasino** [comment]
The 2.1.0.0 release notes say this has been fixed: 
"Add /etc/machine-id in the virtual machine. Fixes docker/for-mac#3554."

**Comment by guillaumerose** [comment]
Let me know if you have an issue!
